scripts/ci/tags.yaml

```diff
@@ -50,10 +50,16 @@ bluetooth:
     - zephyr/subsys/bluetooth/
     - zephyr/include/zephyr/bluetooth/
     - zephyr/samples/bluetooth/hci_ipc/
+    - zephyr/soc/nordic/
+    - zephyr/subsys/settings/
+    - zephyr/cmake/
+    - zephyr/share/
```

Comment:
Do we want to keep it ordered alphabetically?

Reply:
Will fix - will sort all tag's dependencies.